_data/simple-icons.json

```diff
@@ -4503,7 +4506,7 @@
         {
             "title": "Jamboard",
             "hex": "F37C20",
-            "source": "https://cdn2.hubspot.net/hubfs/159104/ECS/Jamboard/Approved%20Jamboard%20Brand%20Book.pdf",
+            "source": "https://workspace.google.com/products/jamboard/",
```

Comment:
I'm not sure for this one since there are no vector version in this page.

Reply:
changed to commons wiki. Plz check and resolve.